src/pages/staticPages/userManual/UserManualMain.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect } from 'react';
+import MainHeading from '../../../common/components/mainHeading/MainHeading';
+import { useTranslation } from 'react-i18next';
+import styles from '../StaticContent.module.scss';
+import { BREADCRUMBS, useBreadcrumbs } from '../Breadcrumbs';
+import Text from '../../../common/components/text/Text';
+
+import { Box } from '@chakra-ui/react';
+import { IconEnvelope, IconPhone, Link } from 'hds-react';
+
+const UserManualMain: React.FC = () => {
+  const { t } = useTranslation();
+  const { setBreadcrumbs } = useBreadcrumbs();
+
+  useEffect(() => {
+    const updateBreadcrumbs = () => setBreadcrumbs([BREADCRUMBS.manual]);
+
+    updateBreadcrumbs();
+  }, [setBreadcrumbs]);
+
+  return (
+    <>
+      <MainHeading spacingBottom="xl">{t('staticPages:manualPage:main:heading')}</MainHeading>
+      <div className={styles.content}>
+        <p>{t('staticPages:manualPage:main:text')}</p>
+        <Box
+          as="section"
+          backgroundColor="var(--color-black-5)"
+          padding="var(--spacing-l)"
+          marginTop="var(--spacing-xl)"
+        >
+          <Text styleAs="h2" tag="h2">
+            {t('staticPages:manualPage:main:contact')}
+          </Text>
+          <br />
+          <Box as="p" marginBottom="var(--spacing-2-xs)">
+            <strong>{t('staticPages:manualPage:main:haitatonContact')}</strong>
+          </Box>
+          <Link
+            style={{ display: 'block' }}
+            iconLeft={<IconEnvelope />}
+            href="mailto:haitatontuki@hel.fi"
+          >
+            haitatontuki@hel.fi
+          </Link>
+          <br />
+          <Box as="p" marginBottom="var(--spacing-2-xs)">
+            <strong>{t('staticPages:manualPage:main:cableServiceContact')}</strong>
+          </Box>
+          <Link
+            style={{ display: 'block' }}
+            external
```

Comment:
Should there be `openInNewTab` prop here as well?

Reply:
Added openInNewTab and aria-lable props